src/sass/static/pages.scss

```diff
@@ -1612,6 +1612,38 @@ body #not_authenticated_financial ul.checked > li {
     & .ui-accordion .ui-accordion-icons {
         padding-right: 1.2em;
     }
+    & #account-closure-dialog {
+        display: none;
+        left: 0;
+
+        #account-closure-dialog-content {
+            width: 496px;
```

Comment:
it's not a good idea to hardcode pixel width and height since responsiveness will be broken (mobile view etc). if you need to hardcode values, it's better to use either rem or percentage values.

Reply:
please test this entire dialog in different browser sizes and update any css values to be responsive that need it accordingly